ulib/fs/ulibfs.fsproj

```diff
@@ -1,46 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>e3e96b71-22ab-4518-b08e-9c55c4d256db</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>ulibfs</RootNamespace>
-    <AssemblyName>ulibfs</AssemblyName>
+    <TargetFramework>netstandard2.0</TargetFramework>
     <OtherFlags>--nowarn:0086 --mlcompatibility --nologo</OtherFlags>
-    <UseStandardResourceNames>true</UseStandardResourceNames>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <Name>ulibfs</Name>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
```

Comment:
@kant2002 Previously this was responsible for installing ulibfs.dll into bin directory.

Reply:
Aah. Then I resurrect parts of it.